src/renderer/components/Footer.tsx

```diff
@@ -26,8 +31,40 @@ export const BasicFooter = styled.div`
 `;
 
 export const Footer: React.FC = () => {
+  const { showInfo, showError } = useToasts();
+  const { dolphinService } = useServices();
+  const { checkForAppUpdates } = useAppUpdate();
+  const { updateDolphin } = useDolphinActions(dolphinService);
+  const [checkingForUpdates, setCheckingForUpdates] = React.useState(false);
+  const checkForUpdatesHandler = React.useCallback(async () => {
+    setCheckingForUpdates(true);
+    try {
+      showInfo("Checking for updates...");
+      await checkForAppUpdates();
+      await updateDolphin();
+    } catch (err) {
+      window.electron.log.error(err);
+      showError("Failed to get updates");
+    }
+    setTimeout(() => setCheckingForUpdates(false), 10000);
+  }, [checkForAppUpdates, updateDolphin, showInfo, showError]);
+
   return (
     <Outer>
+      <Button
```

Comment:
Instead of introducing yet another button, let's repurpose the Slippi logo in the top right for checking for updates. I doubt anyone has ever clicked that intentionally.